### PR TITLE
Force hardware acceleration, recalculate layout in another thread

### PR DESCRIPTION
**Pull request by cicloon**
This improve the rendering performance severely on devices with Android version less than 5.0. 

Take a look @lurecas, I'm sure there are more things to improve here.

### CODE REVIEW

**Comment by cicloon** [comment]
I think this is the next thing we should review: https://github.com/mapsplugin/cordova-plugin-googlemaps/issues/835

File: src/android/plugin/google/maps/GoogleMaps.java [hunk 1]
```diff
@@ -192,9 +192,9 @@ public void run() {
             e.printStackTrace();
           }
        */
-        if (Build.VERSION.SDK_INT >= 21 || "org.xwalk.core.XWalkView".equals(view.getClass().getName())){
```
Comment (lurecas) [inline src/android/plugin/google/maps/GoogleMaps.java:195]: I'm trying to figure out why they would do this...but I dunno...
Have we tested this with several devices? It's like they are checking that the webview is one of the standard class ( not Crosswalk!)

**Comment by lurecas** [comment]
If this works, all :pear: from me!

**Comment by lurecas** [comment]
And yes, @cicloon I also think we will have to give a look at some marker clustering options. We are having an increasingly number of cars (especially in LATAM) and I think that it doesn't give you much info to see a lot of icons one top of another. But this need some design input
What do you guys think @ciscou @ropa03 ?

**Comment by ciscou** [comment]
:fork_and_knife:DD

File: src/android/plugin/google/maps/GoogleMaps.java (hunk 1 above)
Comment (wf9a5m75) [inline src/android/plugin/google/maps/GoogleMaps.java:195]: Because XWalkView class does not have `setLayerType` method.

**Comment by hirbod** [comment]
"org.xwalk.core.XWalkView".equals(view.getClass().getName())" check was added by me and it is important in order to work with Crosswalk. Your fork should not work with crosswalk on specific version anymore

**Comment by cicloon** [comment]
Hi @Hirbod, with crosswalk version 16 the XWalkView class isn't "org.xwalk.core.XWalkView" anymore. And we only use crosswalk to package our app, that's why I deleted that line at all.

**Pull request merged**